.github/workflows/node-ci.yml

```diff
@@ -58,35 +47,54 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-    # Collect and record coverage
-    - name: Run Jest with coverage
-      run: yarn coverage
-    # - name: Upload Coverage
-    #   # We don't bother collecting a record of coverage for dependabot changes
-    #   if: ${{ github.actor != 'dependabot[bot]' }}
-    #   uses: codecov/codecov-action@v2
-    #   with:
-    #     token: ${{ secrets.CODECOV_TOKEN }}
-    #     files: ./coverage/coverage-final.json
-    #     fail_ci_if_error: true
+    # Run tests for our target matrix
+    - name: Run Jest (all files)
+      if: |
+        contains(steps.changed.outputs.added_modified, 'jest.config.js') ||
+        contains(steps.changed.outputs.added_modified, 'yarn.lock') ||
+        contains(steps.changed.outputs.added_modified, 'test.config.js') ||
+        contains(steps.changed.outputs.added_modified, 'test.transform.js')
```

Comment:
I'm assuming the full path isn't needed here?

Reply:
I don't think so.  This was already there, but part of another check.